ctapipe/containers.py

```diff
@@ -484,6 +484,9 @@ class SimulatedCameraContainer(Container):
 
     container_prefix = ""
 
+    true_image_sum = Field(
+        np.nan, "Sum of the detected Cherenkov photons in the camera"
```

Comment:
maybe add a type here or a note in the description to remind us that it is a scalar not a sum in time?

Reply:
"Sum of photons over the camera" is pretty clear on that it is a scalar, right?

Reply:
I read it as the sum in time, which is why I was confused at first (e.g. the sum for each pixel). Here it's the temporal and spatial sum.

Reply:
Maybe we should call it true_intensity? Since intensity is what we call the image sum later.